odoo/tests/common.py

```diff
@@ -537,8 +537,23 @@ def executable(self):
 
         raise unittest.SkipTest("Chrome executable not found")
 
+    def _spawn_chrome(self, cmd):
+        pid = os.fork()
```

Comment:
`fork()` won't work on Windows but I guess it's fine because `executable()` will raise first. For consistency we might still want to move the posix test at the top and return early for non-posix. What do you think?

Reply:
I agree